src/ngraph/runtime/cpu/cpu_external_function.hpp

```diff
@@ -72,18 +74,40 @@ namespace ngraph
 
             using OpMap = std::unordered_map<std::type_index, OpFunction>;
 #endif
+            struct TensorTracerAttributes
```

Comment:
Follow the coding standards.

Reply:
I don't understand, it's like OpAttributes there.
Define C++ class or type names with CamelCase.

Reply:
The member names should be `m_number_of_elements`, `m_shape` and `m_element_type`